undefined_syms.txt

```diff
@@ -1,12 +1,15 @@
 gGameStatusPtr = 0x8007419C;
 gUIStatus = 0x8010EF58;
 gPlayerData = 0x8010F290;
+gPlayerActionState = 0x8010F07C;
+gPlayerAnimation = 0x8010F080;
 gPlayerStatus = 0x8010EFC8;
 gItemTable = 0x800878E0;
 gWorldScriptList = 0x802DA490;
 gBattleScriptList = 0x802DA690;
 gCurrentScriptListPtr = 0x802DA890;
 gGlobalTimeSpace = 0x802D9CA8;
+gPlayerNpc = 0x802D9D20;
```

Comment:
I think this should be called `gPlayerNpcPtr` since we've discovered it's actually a pointer